web_src/js/components/RepoContributors.vue

```diff
@@ -88,6 +90,9 @@ export default {
   methods: {
     sortContributors() {
       const contributors = this.filterContributorWeeksByDateRange();
+      const min = new Date(this.xAxisMin).toISOString().split('T')[0];
+      const max = new Date(this.xAxisMax).toISOString().split('T')[0];
```

Comment:
```suggestion
      const min = dayjs(this.xAxisMin).format('YYYY-MM-DD');
      const max = dayjs(this.xAxisMax).format('YYYY-MM-DD');
```

Should work the same and reads a bit better to me.